juju/juju
# Use resource tags to add to labels on deployed pods etc

## Description of change

Add the content of the juju resource tags to the labels on pods, secrets, and other resources created in k8s. This mirrors the functionality for cloud based models.

Also, remove code which used a deployment controller for the operator as that's no longer supported.

## QA steps

deploy a k8s model
add some resource tags
deploy a k8s charm
inspect the pods and see that the labels are correctly set
